JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -0,0 +1,93 @@
+http://www.w3.org/ns/ssn/systems/SurvivalRange,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,Survival range
+http://www.w3.org/ns/ssn/systems/OperatingRange,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,Operating range
+http://www.w3.org/ns/ssn/systems/SystemCapability,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,System capability
+http://www.w3.org/ns/ssn/systems/SystemProperty,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,System property
+http://www.w3.org/ns/ssn/systems/Frequency,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Frequency
+http://www.w3.org/ns/ssn/systems/Accuracy,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Accuracy
+http://www.w3.org/ns/ssn/systems/MeasurementRange,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Measurement range
+http://www.w3.org/ns/ssn/systems/Precision,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Precision
+http://www.w3.org/ns/ssn/systems/Resolution,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Resolution
+http://www.w3.org/ns/ssn/systems/Sensitivity,Class,http://www.w3.org/ns/ssn/systems/SystemProperty,IS-A,,,,,http://www.w3.org/ns/ssn/systems/,Sensitivity
+http://www.w3.org/ns/ssn/systems/Condition,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,Condition
+http://www.w3.org/ns/ssn/systems/OperatingProperty,Class,,,,,,,http://www.w3.org/ns/ssn/systems/,Operating property
+Setpoint,Class,,,,,,"The desired or target value for an essential variable, or process value (PV) of a control system.",https://www.theworldavatar.com/kg/ontodevice/,Setpoint
+InOutState,Class,https://saref.etsi.org/core/State,IS-A,,,,The current state of something on whether it is inside or outside of a space or object.,https://www.theworldavatar.com/kg/ontodevice/,In out state
+PowerFactor,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,"The ratio of working power, measured in kilowatts (kW), to apparent power, measured in kilovolt amperes (kVA).",https://www.theworldavatar.com/kg/ontodevice/,Power factor
+ScreenBrightness,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,The brightness of the screen of a device measured as a ratio.,https://www.theworldavatar.com/kg/ontodevice/,Screen brightness
+http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Ratio
+Noise,Class,https://www.theworldavatar.com/kg/ontouom/SoundPressureLevel,IS-A,,,,"A sound, especially one that is loud or unpleasant or that causes disturbance. ",https://www.theworldavatar.com/kg/ontodevice/,Noise
```

Comment:
While I have no problem with defining noise as "a sound, especially one that is loud or unpleasant or that causes disturbance", I don't think it's right to define that as a subclass of SoundPressureLevel. I think noise defined like that can _have_ a sound pressure level (as a property). Please adjust accordingly.